packages/dialtone-css/lib/build/less/utilities/colors.less

```diff
@@ -13,110 +13,17 @@
 //
 //  $$  TEXT COLORS
 //  ----------------------------------------------------------------------------
-.d-fc-primary { color: var(--dt-color-foreground-primary) !important; }
-.d-fc-primary-inverted { color: var(--dt-color-foreground-primary-inverted) !important; }
-.d-fc-secondary { color: var(--dt-color-foreground-secondary) !important; }
-.d-fc-secondary-inverted { color: var(--dt-color-foreground-secondary-inverted) !important; }
-.d-fc-tertiary { color: var(--dt-color-foreground-tertiary) !important; }
-.d-fc-tertiary-inverted { color: var(--dt-color-foreground-tertiary-inverted) !important; }
-.d-fc-muted { color: var(--dt-color-foreground-muted) !important; }
-.d-fc-muted-inverted { color: var(--dt-color-foreground-muted-inverted) !important; }
-.d-fc-placeholder { color: var(--dt-color-foreground-placeholder) !important; }
-.d-fc-placeholder-inverted { color: var(--dt-color-foreground-placeholder-inverted) !important; }
-.d-fc-disabled { color: var(--dt-color-foreground-disabled) !important; }
-.d-fc-disabled-inverted { color: var(--dt-color-foreground-disabled-inverted) !important; }
-.d-fc-success { color: var(--dt-color-foreground-success) !important; }
-.d-fc-success-inverted { color: var(--dt-color-foreground-success-inverted) !important; }
-.d-fc-success-strong { color: var(--dt-color-foreground-success-strong) !important; }
-.d-fc-success-strong-inverted { color: var(--dt-color-foreground-success-strong-inverted) !important; }
-.d-fc-warning { color: var(--dt-color-foreground-warning) !important; }
-.d-fc-warning-inverted { color: var(--dt-color-foreground-warning-inverted) !important; }
-.d-fc-error { color: var(--dt-color-foreground-critical) !important; }
-.d-fc-error-strong { color: var(--dt-color-foreground-critical-strong) !important; }
-.d-fc-critical { color: var(--dt-color-foreground-critical) !important; }
-.d-fc-critical-inverted { color: var(--dt-color-foreground-critical-inverted) !important; }
-.d-fc-critical-strong { color: var(--dt-color-foreground-critical-strong) !important; }
-.d-fc-critical-strong-inverted { color: var(--dt-color-foreground-critical-strong-inverted) !important; }
```

Comment:
Should we have just kept them here instead of generating with postcss?

example:
```
.d-fc-primary { 
  color: hsl(var(--dt-color-foreground-primary-h) var(--dt-color-foreground-primary-s) var(--dt-color-foreground-primary-l) / var(--fco, var(--dt-color-foreground-primary-a))) !important; 
}
```

On the one hand, doing them in postcss is more automated, on the the other hand, it is much harder to understand what's going on... Generally we use the pattern that all util classes are defined in the less files and then postcss is used to generate hover/focus variants. It's kind of weird to not do that just in this one case for color. What do you think?

Reply:
In fact, we were already generating most color utility classes on PostCSS, we were missing the semantic `font-color`, `border-color` and `background-color` utility classes.

We can move all of them to either LESS or PostCSS but I completely agree, having them on PostCSS is more automated but pretty obscure.

Having them in LESS is way easier to find and fix something that is broken.

We're already generating all this utility classes.

![Screenshot 2024-10-03 at 10 01 14 p m](https://github.com/user-attachments/assets/e7fa71d4-3164-4ee0-ae5c-6dd9f92c8dd4)

Certainly we don't want to come back to what we had years ago, maybe we just need to add some documentation to make it easier to find what we're generating on PostCSS and what on LESS.